exercises/gagik_vardanyan/ex02_oo_basics/CMakeLists.txt

```diff
@@ -18,11 +18,27 @@ include ( CommonSetup )
 
```

Comment:
Excellent. These changes look like they should work. I have to admit that I haven't checked any compiler outputs yet.